pkg/kubelet/container/runtime.go

```diff
@@ -472,6 +472,9 @@ type Mount struct {
 	Propagation runtimeapi.MountPropagation
 	// Image is set if an OCI volume as image ID or digest should get mounted (special case).
 	Image *runtimeapi.ImageSpec
+	// ImageSubPath is set if an image volume sub path should get mounted. This
+	// field is only required if the above Image is set.
```

Comment:
It is an optional field. It can be set only when the above Image is set.

Reply:
Is there anything you'd like to have changed here? :upside_down_face:

Reply:
Sorry, I misread it and thought this field was required if the image was set.